src/in/shick/diode/comments/CommentsListActivity.java

```diff
@@ -1669,6 +1670,22 @@ public boolean onContextItemSelected(MenuItem item) {
             resetContextInfo();
             getNewDownloadCommentsTask().execute(Constants.DEFAULT_COMMENT_DOWNLOAD_LIMIT);
             return true;
+            
+        case Constants.SHARE_COMMENTS:
+            Intent intent2 = new Intent();
+            intent2.setAction(Intent.ACTION_SEND);
+            intent2.setType("text/plain");
+
+            intent2.putExtra(Intent.EXTRA_SUBJECT, getOpThingInfo().getTitle());
+            intent2.putExtra(Intent.EXTRA_TEXT, "https://reddit.com/r/" + mSubreddit + "/" + mThreadId );
```

Comment:
The base url constant (https://github.com/zagaberoo/diode/blob/master/src/in/shick/diode/common/Constants.java#L358) would be a good fit in this line.